wagtail/test/testapp/migrations/0001_initial.py

```diff
@@ -3855,6 +3899,26 @@ class Migration(migrations.Migration):
                 verbose_name="Tags",
             ),
         ),
+        migrations.CreateModel(
+            name="RevisableGrandChildModel",
+            fields=[
+                (
+                    "revisablechildmodel_ptr",
+                    models.OneToOneField(
+                        auto_created=True,
+                        on_delete=django.db.models.deletion.CASCADE,
+                        parent_link=True,
+                        primary_key=True,
+                        serialize=False,
+                        to="tests.revisablechildmodel",
+                    ),
+                ),
+            ],
+            options={
+                "abstract": False,
+            },
+            bases=("tests.revisablechildmodel",),
+        ),
```

Comment:
eek, can't figure out what's going on here :-)

Please create _new_ migrations on top of the 0001_squashed_0073 one. If you're editing or deleting the existing ones, you're almost certainly doing something wrong.

Reply:
Ah. The `testapp` migrations were zapped within this MR months ago (after checking with Karl), though I've rewritten the commit history for this MR so that it's now consistent with https://github.com/wagtail/wagtail/pull/8783.